doc/r-plugin.txt

```diff
@@ -506,18 +506,17 @@ The plugin commands that send code to R Console are the most commonly used. If
 the code to be sent to R has a single line it is sent directly to R Console,
 but if it has more than one line (a selection of lines, a block of lines
 between two marks, a paragraph etc) the lines are written to a file and the
-plugin sends to R the command to source the file. You should type quickly
-<LocalLeader>d to send to R Console the line currently under the cursor. If
-you want to see what lines are being sourced when sending a selection of
-lines, you should do either <LocalLeader>se or <LocalLeader>sa instead of
-<LocalLeader>ss.
+plugin sends to R the command to source the file. To send to R Console the
+line currently under the cursor you should type <LocalLeader>d. If you want to
+see what lines are being sourced when sending a selection of lines, you can
+use either <LocalLeader>se or <LocalLeader>sa instead of <LocalLeader>ss.
 
 If the cursor is over the header of an R chunk with the `child` option (from
 Rnoweb, RMarkdown or RreStructuredText document), and you use one of the
 commands that send a single line of code to R, then the plugin will send to R
 the command to knit the child document.
 
-After the commands that send, sweave or knit the current buffer, Vim will save
```

Comment:
We may rewrite the sentence to make it clearer, but just removing the comma is incorrect because my intention was to list three commands: (1) send the buffer; (2) sweave the buffer; (3) knit the buffer.